doc/README.yaml.md

```diff
@@ -0,0 +1,78 @@
+
+# README for Open-CE Environment Files
+
+
+This README documents the YAML files used by the build scripts
+[build_env.py](README.build_env.md)
+and [build_feedstock.py](README.build_feedstock.md).
+Please refer to the respective README files for each of these two scripts.
+
+Most likely you will want to simply use the Open-CE environment files provided within the
+open-ce repository as they are. These files have already been customized and
+verified to work. However, if you want to debug or do your own customizations,
+you can edit them or create your own YAML files for use by these scripts. If you
+do this, you will likely want to start by copying one of the existing files and
+making local edits as desired.
+
+
+# Open-CE Environment File
```

Comment:
There are several more possible fields in an Open-CE Environment file that aren't listed here. You can see a full list of them here: https://github.com/open-ce/open-ce/blob/master/open-ce/env_config.py#L32. The currently supported fields are:

packages:
    feedstock:
    recipes:
    channels:
    git_tag:
imported_envs:
git_tag_for_env:
external_dependencies:
channels:

Reply:
I agree, let's put this list at the top with quick explanations, then we can go into detail on each below like you already do with a few of them.

Reply:
Done for all with exception of `external_dependencies` -- which I'm not seeing cases of how we use this anywhere within any builds -- only for a specific use within the open-ce/tests.  (So, am not sure how to reflect this one in a useful example).

Reply:
`external_dependencies` are the names of conda packages that will not be built, but will be added to the conda environment files that are generated by calling `build_env`.